src/JsonRpc.Host/JsonRpcMiddleware.cs

```diff
@@ -65,7 +83,6 @@ internal class JsonRpcMiddleware
                 responseString = await processor.ProcessAsync(requestString, service);
                 diagnostics.ProcessTime = watch.ElapsedMilliseconds - diagnostics.ReadTime;
                 response.Headers.Add("Content-Type", "application/json");
-                response.Headers.Add("X-ProcessingTime", diagnostics.ProcessTime.ToString());
```

Comment:
Yeah. Removed because this is sort of an information leak. Does not seem to be of any importance to frontend and yet may be used for some malicious purpose (e.g. timing attack).